frontend/src/lib/forms/RadioButtonGroup.svelte

```diff
@@ -44,7 +45,7 @@
   {#if description}
     <label for={id} class="label pb-0">
       <span class="label-text-alt">
-        <Markdown md={description} plugins={[{ renderer: { a: NewTabLinkRenderer } }]} />
+        <Markdown md={description} plugins={[{ renderer: { a: NewTabLinkRenderer as Component<any> } }]} />
```

Comment:
I didn't realize how many places we were using the Markdown component, since we always override the renderer is there a better way to handle that? maybe our own component or something else?

Reply:
Don't know, let's talk about it at the meeting.

Reply:
we'll turn this into a component where the new tab links are optional, this will be done in another PR once this is merged

Reply:
Done in commit 5f372111694, because I forgot that we were planning to do this in another PR.